src/ITensors.jl

```diff
@@ -202,11 +177,24 @@ include("physics/site_types/tj.jl")
 include("physics/site_types/qudit.jl") # EXPERIMENTAL
 include("physics/site_types/boson.jl") # EXPERIMENTAL
 include("physics/fermions.jl")
-include("physics/autompo/matelem.jl")
-include("physics/autompo/qnmatelem.jl")
-include("physics/autompo/opsum_to_mpo_generic.jl")
-include("physics/autompo/opsum_to_mpo.jl")
-include("physics/autompo/opsum_to_mpo_qn.jl")
+
+#####################################
+# MPS/MPO
+#
+include("ITensorMPS/ITensorMPS.jl")
+@reexport using .ITensorMPS
+
+# Only include iterativesolvers.jl after ITensorMPS
+# since it defines orthogonalize! which could
+# conflict with definition in ITensorMPS
+import .ITensorMPS: orthogonalize!
```

Comment:
This doesn't seem right to me. I have a few comments about this:
1. Can we move `iterativesolvers.jl` to `ITensorMPS`? It is only used by the MPS code anyway, and ultimately we should just deprecate it in favor of KrylovKit.jl so we may as well start that process now by moving it out of the base ITensors.jl codebase.
2. I think we should rename the `orthogonalize!` function use inside the Krylov solver to something else, like `krlyov_orthogonalize!`. It is conceptually unrelated from the MPS `orthogonalize!` function so we may as well just give them different names and avoid future namespace issues.